### ISSUE TITLE
Seems like the documentation regarding GID is wrong?

### ISSUE THREAD

**Issue by Naville** [aria2/aria2#1276]
Hi:
The documentation regarding GID says

> Set GID manually. aria2 identifies each download by the ID called GID. The GID must be hex string of 16 characters, thus  [0-9a-zA-Z] are allowed and leading zeros must not be stripped. The GID all 0 is reserved and must not be used. The GID must be unique, otherwise error is reported and the download is not added. This option is useful when restoring the sessions saved using --save-session option. If this option is not used, new GID is generated by aria2.

In my test cases I got some error logged by aria2:
```
10/13 08:38:58 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 eu5d4zsec34sockw is invalid for GID.

10/13 08:38:58 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 fm1alm2cvnbnr178 is invalid for GID.

10/13 08:38:58 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 nryb7uwlh9p4tyxy is invalid for GID.

10/13 08:38:58 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 kibqkojtvdtsx3q3 is invalid for GID.
[DL:12KiB][#db11c4 2.9MiB/7.7MiB(38%)][#2e68e1 3.1MiB/4.4MiB(72%)]                                                                                                                               
10/13 08:39:08 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 eu5d4zsec34sockw is invalid for GID.

10/13 08:39:08 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 fm1alm2cvnbnr178 is invalid for GID.

10/13 08:39:08 [ERROR] Exception caught
Exception: [download_helper.cc:114] errorCode=1 nryb7uwlh9p4tyxy is invalid for GID.
```

Seems like these invalid GIDs all contains [g-zG-Z], since the documentation also mentioned that GID needs to be a hex string, I guess that part should be ``thus  [0-9a-fA-F] are allowed`` ?

**Comment by Naville** [comment]
Taking a look at my saved session file:

```
https://******.pdf	
 gid=db11c450d2d111e4
 timeout=20
 dir=**********
 out=******.pdf	
 allow-overwrite=false
 continue=true
 auto-file-renaming=false
 remote-time=true
 max-connection-per-server=4
```
And this GID contains only [0-9a-fA-F]

**Comment by tatsuhiro-t** [comment]
I think documentation is wrong.  It should be [0-9a-fA-F]

**Comment by tatsuhiro-t** [comment]
Fix committed via 7e5757c2
